tpkg/common.sh

```diff
@@ -250,6 +250,33 @@ wait_server_up_or_fail () {
 	done
 }
 
+# $1: zone
+# $2: serial to be expected
+# $3: server to query
+# $4: port
+# $5: # times to try (# seconds dig is ran)
+wait_for_soa_serial () {
+	TS_START=`date +%s`
+	for i in `eval echo {1..$5}`
```

Comment:
The `eval` call is not needed here, it is better not to use it, ``` for i in `echo {1..$5}` ```

Reply:
For portability to non-bash, this should be ``` for i in `seq 1 $5` ```

Reply:
Changed it